inventory_management_system_api/models/catalogue_item.py

```diff
@@ -44,9 +27,7 @@ class CatalogueItemIn(BaseModel):
     name: str
     description: str
     properties: List[Property] = []
-    # pylint: disable=fixme
-    # TODO - Change from manufacturer to manufacturer id
-    manufacturer: Manufacturer
+    manufacturer_id: str
```

Comment:
Its type should be `CustomObjectIdField`. Could you please also update the repo unit tests accordingly?
```suggestion
    manufacturer_id: CustomObjectIdField
```

Reply:
@MatteoGuarnaccia5 There are some repo unit tests that have not been updated to return the `manufacturer_id` as `CustomObjectId` object. Have a look at the `test_update` test and some of the `test_list_...` tests.